### PR TITLE
chore: publish v1.2.0

### PR DESCRIPTION
**Pull request by dshukertjr**

### CODE REVIEW

**Comment by Vinzent03** [comment]
I read on the Discord that `listUsers` is actually paged so it doesn't return all users. See [gotrue-js issue](https://github.com/supabase/gotrue-js/issues/538). Do we want to fix that before release or remove the method or keep the bug for now?

**Comment by dshukertjr** [comment]
@Vinzent03 Great find! I was not aware of this at all. I think it's okay to release this for now, and let's implement a feature to get different page once it is implemented on js library!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## [1.2.0]
+
+- feat: add `createUser()`, `deleteUser()`, and `listUsers()` to admin methods.
```
Comment (Vinzent03) [inline CHANGELOG.md:3]: What about adding a link to the pr?

**Review by Vinzent03 — CHANGES_REQUESTED** [review]
Okay

File: CHANGELOG.md (hunk 1 above)
Comment (dshukertjr) [reply CHANGELOG.md:3]: Great catch! Let me add them right now!

**Review by Vinzent03 — APPROVED** [review]

**Comment by j4w8n** [comment]
> I read on the Discord that `listUsers` is actually paged so it doesn't return all users. See [gotrue-js issue](https://github.com/supabase/gotrue-js/issues/538). Do we want to fix that before release or remove the method or keep the bug for now?

@Vinzent03 this is now a thing in gotrue-js

https://github.com/supabase/gotrue-js/pull/544

Not yet included in the supabase-js client

**Comment by Vinzent03** [comment]
@j4w8n I've already implemented it in #109, but you are right it's not jet released. We might bundle it with #110, but I'm not sure. What do you think @dshukertjr ?

**Comment by dshukertjr** [comment]
Oh yeah sorry. It slipped by with all the launch week chaos! Will create a PR for a new version launch right now!

**Comment by dshukertjr** [comment]
@j4w8n A new version containing the update has been published!

**Comment by j4w8n** [comment]
@dshukertjr yep in gotrue-js. They just haven't bumped the version in supabase-js yet.

**Comment by dshukertjr** [comment]
@j4w8n You mean gotrue-dart and supabase-flutter, correct? Yup, even though supabase-flutter has not been updated, you can actually run `dart pub upgrade gotrue`. This will update your pubspec.lock and therefore upgrade the underlying gotrue version used in your app!

**Comment by j4w8n** [comment]
@dshukertjr roger that. I'm a js guy. Just remember Vinzent mentioning this feature a while back, and y'all wanted to wait until the js version was out.

**Comment by Vinzent03** [comment]
@j4w8n Oh I remember. Yeah I subscribed to the pr after you sent it in the Discord so I was aware of, but thanks for taking care!

**Pull request merged**